docs/kmesh_compile-zh.md

```diff
@@ -137,6 +137,62 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
   kmesh-0.0.1           latest              e321b18d5fee        4 hours ago         675MB
   ```
 
+### docker image [兼容模式](../build/docker/README.md)构建
```

Comment:
I donot see the difference with L92, can you explain

Reply:
In L92, images are built in RPM mode. In this example, online compilation (images in compatible mode) is built using kmesh source code. The main difference is that rpm is replaced with kmesh source code. The start_kmesh.sh script contains adaptive compilation and installation actions. Other actions at the user construction level are the same.

Reply:
I see both use the same dockerfiler `docker build -f kmesh.dockerfile -t kmesh-0.0.1 .`